README.md

```diff
@@ -62,10 +67,27 @@ Outstanding issues and requests can be tracked and visualised on our Github Proj
 [Documentation](https://github.com/Mafia-Hands/Mafia/projects/3)
 
 ## Contributing
-If you would like to contribute to this project, please make sure to follow our [Contributing-Guidlines](https://github.com/Mafia-Hands/Mafia/wiki/Contributing-Guidelines).
+
+If you would like to contribute to this project, please make sure to follow our [Contributing-Guidlines](https://github.com/Mafia-Hands/Mafia/wiki/Contributing-Guidelines)s.
+
+## Authors and acknowledgement
+
+This project was coordinated by and for the University of Auckland course SOFTENG 701.
+
```

Comment:
Probably add something some which says we are thanking these two entities below. Right now they are just there under acknowledgement but its weird reading it.